extensions/amp-script/0.1/test/integration/test-amp-script.js

```diff
@@ -88,9 +88,7 @@ describe
 
           // Give mutations time to apply.
           yield browser.wait(100);
-          yield poll('terminated', () => {
-            return element.classList.contains('i-amphtml-broken');
-          });
+          yield poll('dropped', () => {});
```

Comment:
These last two lines don't seem to do anything anymore. Let's verify that `button#hello` noop'ed?

Reply:
The `browser.wait()` is definitely pointless, but shouldn't the polling for `dropped` still be effective? My understanding is that it will wait for the the word "dropped" to be emitted to the console, and in case of a timeout it throws.

Reply:
I misunderstood what `poll` did...nevermind. fixing it